styleguide/guides/issues.rst

```diff
@@ -0,0 +1,172 @@
+📝 Issue Tracking
+=================
+
+.. toctree::
+   :maxdepth: 1
+   :hidden:
+
+   issues/security
+   issues/bug
+   issues/feature
+   issues/refactoring
+   issues/documentation
+
+Summary
++++++++
+Issue tracking is a great tool to track pending work and improve the overall communication within a project.
+But similar to every other tool, it needs to be used with care.
+
+    | "A fool with a tool is still a fool"
+    | -- Grady Booch
+
+This guide will try to help you to get the best out of the issue tracking for you, your teammates and the project(s)
+you are working on, by answering the following questions:
+
+* When to create an issue?
+* What makes a good issue?
+* How to maintain an issue?
+* How to maintain a backlog?
```

Comment:
Very ambitious :)
For me looks like this would be a topic for the whole team, and is not only Python specific....

Reply:
It surely is!